x-pack/plugins/index_lifecycle_management/public/sections/edit_policy/components/min_age_input.js

```diff
@@ -39,26 +51,39 @@ const MinAgeInputUi = props => {
       <EuiFlexItem style={{ maxWidth: 188 }}>
         <EuiFormRow hasEmptyLabelSpace>
           <EuiSelect
-            aria-label={intl.formatMessage({
-              id: 'xpack.indexLifecycleMgmt.editPolicy.minimimAgeUnitsAriaLabel',
-              defaultMessage: '{phaseUpper} phase after units',
-            }, { phaseUpper: `${phase.charAt(0).toUpperCase()}${phase.slice(1)}` })}
+            aria-label={intl.formatMessage(
+              {
+                id: 'xpack.indexLifecycleMgmt.editPolicy.minimimAgeUnitsAriaLabel',
+                defaultMessage: '{phaseUpper} phase after units',
+              },
+              { phaseUpper: `${phase.charAt(0).toUpperCase()}${phase.slice(1)}` }
```

Comment:
I wonder if we want the terms "Hot", "Warm", etc to be localizable. Perhaps something we can ask the localization team to look at later. It might turn out that the best approach will be to create individual messages per phase and store them in a map, and then select the correct message here based on the phase.